### PR TITLE
add detailed implementation with error for petstore server

### PR DESCRIPTION
**Pull request by ArcturusZhang**
This PR implements the server operations for the `Pets` interface in petstore project.

### CODE REVIEW

File: petstore/servers/aspnet/implemented_apis.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+### `Pets.get`
+
+- Endpoint: `/pets/{petId}`
+
+- Returns:
+
+| petId | HTTP Code | Response body |
+| :-- | :-- | :-- |
+| 0 <= petId < 10 | 200 | `{ "id": petId, "age": 5, "name": "Kiwi", "ownerId": 5 }` |
+| others | 400 | `{ "code": 0, "message": "Invalid petId" }` |
```
Comment (lirenhe) [inline petstore/servers/aspnet/implemented_apis.md]: is it better to return 404 in this case?

File: petstore/clients/dotnet/tests/PetsClientTests.cs [hunk 2]
```diff
@@ -1,12 +1,22 @@
...
 namespace PetStore.Tests
 {
     public class PetsClientTests
     {
+        private Pets _petsClient;
+
+        [OneTimeSetUp]
+        public void Setup()
+        {
+            _petsClient = new PetStoreClient(new Uri("http://localhost:5118")).GetPetsClient();
```
Comment (lirenhe) [inline petstore/clients/dotnet/tests/PetsClientTests.cs:16]: can we put this Uri value into a config file? This is likely to be shared by server, client and tests.

Comment (ArcturusZhang) [reply petstore/clients/dotnet/tests/PetsClientTests.cs:16]: this is only temporary.
I think it is no point that we put it into a config or something.
In the future, there could be some script that assigns this value.

File: petstore/servers/aspnet/implemented_apis.md (hunk 1 above)
Comment (ArcturusZhang) [reply petstore/servers/aspnet/implemented_apis.md]: sure, updated

**Review by lirenhe — APPROVED** [review]

**Pull request merged**